ducktape/tests/test.py

```diff
@@ -52,12 +54,22 @@ def min_cluster_size(self):
         """
         return self.test_context.services.num_nodes()
 
-    def setUp(self):
+    def setup(self):
```

Comment:
I suppose verb form would be `set_up`, but the camelCase (`setUp`, `tearDown`) has bugged me for ages.